lib/DDG/Spice/Tvmaze/Nextepisode.pm

```diff
@@ -9,18 +9,23 @@ spice is_cached => 1;
 spice proxy_cache_valid => "200 301 302 404 1h";
 
 triggers any => 'episode', 'airdate';
+triggers start => 'when does';
 
 spice to => 'http://api.tvmaze.com/singlesearch/shows?q=$1&embed=nextepisode';
 spice wrap_jsonp_callback => 1;
 
 handle query_lc => sub {
     return unless $_;
+        
+    return $4 if $_ =~ /(next|upcoming) (episode|airdate) (in|of|for|from)? ?(.+?)/;
+        
+    return $2 if $_ =~ /(next|upcoming) (.+?) episode/;
 
-    return $4 if $_ =~ /(next|upcoming) (episode|airdate) (in|of|for|from)? ?([a-z0-9 ]+)/;
+    return $1 if $_ =~ /(.+?) (next|upcoming) (episode|airdate)/;
+        
+    return $2 if $_ =~ /(when does) (.+?) (start|come back|come out|air|return) ?(on)?/;
```

Comment:
there's no need to capture `when does` -- you can remove the `()` and change `$2` to `$1`.